lib/scrabble.rb

```diff
@@ -15,4 +13,81 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+
+  def score(word)
+    if word == nil
+      return 0
+    elsif word.is_a? Integer
+      return 0
+    else
+      data = word.upcase.split("")
```

Comment:
Looks like you use the 'something.upcase.split("")' multiple times in your code. Consider pulling that logic into its own method to make your code more dry.